NetworkingProject/TableViewCell/ServiceLayer/StorageService/CoreDataService/CoreDataMovies.swift

```diff
@@ -61,11 +61,11 @@ final class CoreDataMovies: DatabaseProtocol {
         coreDataService.saveContext()
     }
 
-    func getDescription(id _: Int) -> [MovieDetails] {
+    func getDescription(id _: Int) -> [MovieDetail] {
```

Comment:
```suggestion
    func getDescription(id _: Int) -> [MovieDetail] {
```
выборку по 1 фильму надо реализовать смотри у Полины или как у Юры